wiki/Ace_contig_class.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Ace contig class
+title: Handling contigs with the Contig class.
```

Comment:
This is specially about the Ace file format and Ace module, so we need Ace in the title somewhere.